on-boot-script/examples/udm-files/on_boot.d/25-add-cron-jobs

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# store crontab files in /mnt/data/cronjobs/ (you will need to create this foler)
+# this script will re-add them on startup
+
+cp /mnt/data/cronjobs/* /etc/cron.d/
+/etc/init.d/crond restart
```

Comment:
Perhaps a `reload` would be better than a full `restart`. But I haven't tested it.

Reply:
`crond reload` for this distro requires the reloading of a specific file to reload, like
`/etc/init.d/crond reload [cron_file]`
 which would make for a much more complex script.
the crond restart isn't particularly taxing:
`# time /etc/init.d/crond restart
Stopping cron ...done.
Starting cron ... done.
real    0m 1.01s
user    0m 0.01s
sys     0m 0.00s
`